realm-annotations-processor/src/main/java/io/realm/processor/RealmJSonImplGenerator.java

```diff
@@ -28,7 +28,7 @@
 
     private static final String REALM_PACKAGE_NAME = "io.realm";
     private static final String CLASS_NAME = "RealmJsonImpl";
-    private static final String EXCEPTION_MSG = "\"Could not find the generated proxy class for \" + classQualifiedName";
+    private static final String EXCEPTION_MSG = "\"Could not find the generated proxy class for \"";
```

Comment:
Not really. Then I just need to add them to where this is used. But that would probably fit the pattern we are using elsewhere.